.editorconfig

```diff
@@ -14,4 +14,4 @@ ktlint_standard = disabled
 
 # SPDX License Names
 [buildSrc/src/main/kotlin/com/datadog/gradle/plugin/checklicenses/SPDXLicense.kt]
-ktlint_standard_enum-entry-name-case = disabled
\ No newline at end of file
+ktlint_standard_enum-entry-name-case = di   sabled
```

Comment:
that is some wrong change

Reply:
yes...it is al this problems with ktlint that I have to do it manually...I will work on that task right now to fix it in develop